pkgs/applications/networking/irc/weechat/scripts/default.nix

```diff
@@ -1,10 +1,12 @@
-{ callPackage, luaPackages }:
+{ callPackage, luaPackages, python3Packages }:
```

Comment:
this should probably be `python` and in all-packages this should be passed `python = python3;` then use `python.pkgs.callPackage`; but I don't feel strongly enough about that to require a change.